docs/installation.rst

```diff
@@ -25,9 +25,13 @@ Distribution Package
 --------------------
 
 Some distributions might offer a ready-to-use ``borgbackup``
-package which can be installed with the package manager.  As |project_name| is
-still a young project, such a package might be not available for your system
-yet.
+package which can be installed with the package manager.
+
+Those packages may not be up to date with the latest Borg releases
+and problems with those packages should first be reported to the
+operating system package maintainers instead of upstream |project_name|.
+
```

Comment:
it's generally the advice we give to upstream in Debian anyways. i assume here that if the bug report rots in the distro, it will eventually be tricked upstream by the submitter.